File: QViz.NET/Periscope.QViz/JSON/APITestResult.cs
using System.Collections.Generic;

namespace Periscope.QViz.JSON
{
	/// <summary>
	/// JSON Model for Test API Result Details
	/// </summary>
	public class APITestResult
	{
		/// <summary>
		/// Test Result Object associated with the API Test
		/// </summary>
		public TestResult testResult;
		
		/// <summary>
		/// List of API Test Result Objects
		/// </summary>
		public IList<APIResult> apiResults;

		/// <summary>
		/// Default Constructor
		/// </summary>
		public APITestResult()
		{
			this.testResult = new TestResult();
			this.apiResults = new List<APIResult>();
		}
	}
}

File: QViz.Java/src/main/java/com/periscope/qviz/json/APITestResult.java
package com.periscope.qviz.json;

import java.util.ArrayList;
import java.util.List;

/**
 * JSON Model for Test API Result Details
 */
public class APITestResult {

	/**
	 * Test Result Object associated with the API Test
	 */
	public TestResult testResult;

	/**
	 * List of API Test Result Objects
	 */
	public List<APIResult> apiResults;

	/**
	 * Default Constructor
	 */
	public APITestResult() {
		this.testResult = new TestResult();
		this.apiResults = new ArrayList<>();
	}

}
